templates/Customer/show.html.twig

```diff
@@ -1,5 +1,166 @@
 {% extends 'base.html.twig' %}
 
-{% block body %}
-    {{ dump(customer) }}
+{% block title %}
+Customer Details
 {% endblock %}
+
+{% block body %}
+<div class="py-5 text-center">
+    <h2>Customer Details</h2>
+</div>
+
+<div class="container">
+
+    {#Basic Information Section#}
+    <h3>
+        <i class="bi bi-person-fill"></i>
+        Basic Information
+    </h3>
+
+    <div class="row mt-4">
+        <div class="col-md-4">
+            <h6>First Name</h6>
+            <p class="text-break">{{customer.firstName}}</p>
+        </div>
+
+        <div class="col-md-4">
+            <h6>Last Name</h6>
+            <p class="text-break">{{customer.lastName}}</p>
+        </div>
+
+        <div class="col-md-4">
+            <h6>Date of Birth</h6>
+            {{ customer.dateOfBirth|date('Y-m-d') }}
+        </div>
+    </div>
+
+    <div class="row mt-4">
+        <div class="col-md-4">
+            <h6>Email</h6>
+            <p class="text-break">{{customer.email}}</p>
+        </div>
+
+        <div class="col-md-4">
+            <h6>Phone Number</h6>
+            {{customer.phone}}
+        </div>
+    </div>
+    <hr class="my-4">
+
+    {#Home Address Section#}
```

Comment:
Duplicit code blocks. You can create new component and include it here 3 times with different parameters